torch_xla/csrc/torch_util.h

```diff
@@ -23,6 +25,13 @@ static inline at::Tensor CopyTensor(const at::Tensor& ref) {
   return ref.to(ref.options(), /*non_blocking=*/false, /*copy=*/true);
 }
 
+// Same as above, with an additional cast.
+static inline at::Tensor CopyAndCastTensor(const at::Tensor& ref,
```

Comment:
I'd just call this CopyTensor() as well.

Reply:
Done.